docs/instrumentations/java/java.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Java Native Instrumentation"
+sidebarTitle: "Native instrumentation"
+---
+
+## Supported Versions
+
+Odigos uses the official OpenTelemetry instrumentation libraries and SDK components, thus it supports the same Java versions as this project.
+
+- Java Virtual Machine runtime versions **8** and above are supported.
+
+## Traces
+
+Odigos will automatically instrument your Java sources to record and collect spans for distributed tracing, by utilizing the OpenTelemetry Java Official Instrumentation Libraries.
+
+If you use one of the supported logger libraries, the trace id and span id will be auto-injected into your log records allowing signals correlations between logs and traces.
+
+## Instrumentation Libraries
+
+The following Java modules will be auto instrumented by Odigos:
+
+### HTTP Frameworks
+- [`play-ws`](https://central.sonatype.com/artifact/com.typesafe.play/play-ws_3) `versions >=1.0.0 <2.1.0`
+- [`jaxrs`](https://central.sonatype.com/artifact/com.jslsolucoes/jaxrs) `versions >=0.5`
+- [`resteasy-jaxrs`](https://central.sonatype.com/artifact/org.jboss.resteasy/resteasy-jaxrs) `versions >=3.0.0 <4.0.0`
+- [`jersey-server`](https://central.sonatype.com/artifact/com.sun.jersey/jersey-server) `versions >=2.0.0 <3.0.0`
+- [`jersey-container-servlet`](https://central.sonatype.com/artifact/org.glassfish.jersey.containers/jersey-container-servlet) `versions >=2.0.0 <3.0.0`
+- [`spring-web`](https://central.sonatype.com/artifact/org.springframework/spring-web) `versions >=3.1.0`
+- [`spring-webflux`](https://central.sonatype.com/artifact/org.springframework/spring-webflux) `versions >=5.0.0`
+- [`spring-webmvc`](https://central.sonatype.com/artifact/org.springframework/spring-webmvc) `versions >=3.1.0`
+- [`javalin`](https://central.sonatype.com/artifact/io.javalin/javalin) `versions >=5.0.0`
+- [`finatra-http`](https://central.sonatype.com/artifact/com.twitter/finatra-http_2.13) `versions >=2.9.0`
+- [`ktor-server-core`](https://central.sonatype.com/artifact/io.ktor/ktor-server-core) `versions >=2.0.0`
+
+### HTTP Clients
+- [`okhttp`](https://central.sonatype.com/artifact/com.squareup.okhttp/okhttp) `versions >=3.0.0`
+- [`httpasyncclient`](https://central.sonatype.com/artifact/org.apache.httpcomponents/httpasyncclient) `versions >=4.1`
+- [`elasticsearch-rest-client`](https://central.sonatype.com/artifact/org.elasticsearch.client/elasticsearch-rest-client) `versions >=6.4 <7.0`
+- [`elasticsearch-transport`](https://central.sonatype.com/artifact/org.elasticsearch/elasticsearch-transport-wares) `versions >=5.0 <8.0`
+- [`elasticsearch-rest`](https://central.sonatype.com/artifact/org.apache.camel/camel-elasticsearch-rest) `versions >=7.0`
+- [`elasticsearch-java`](https://central.sonatype.com/artifact/co.elastic.clients/elasticsearch-java) `versions >=7.16 <7.17.20`, `versions >=8 <8.10`
```

Comment:
I think all elasticsearch instrumentations are better placed in the database client section. WDYT?